extensions/amp-facebook/1.0/README.md

```diff
@@ -0,0 +1,438 @@
+
+Specifies the tabs to render (i.e., `timeline`, `events`, `messages`). Use a
+comma-separated list to add multiple tabs (e.g., `timeline, events`). By
+default, the Facebook page plugin shows the timeline activity.
+
+##### `data-hide-cover` (optional)
+
+Hides the cover photo in the header. Default value is `false`.
+
+##### `data-show-facepile` (optional)
+
+Shows profile photos of friends who like the page. Default value is `true`.
+
+##### `data-hide-cta` (optional)
+
+Hides the custom call to action button (if available). Default value is `false`.
+
+##### `data-small-header` (optional)
+
+Uses the small header instead. Default value is `false`.
+
+### Preact/React Component
+
+The examples below demonstrate use of the `<BentoFacebook>` as a functional component usable with the Preact or React libraries.
+
+#### Example: Import via npm
+
+[example preview="top-frame" playground="false"]
+
+Install via npm:
+
+```sh
+npm install @ampproject/bento-facebook
+```
+
+```javascript
+import React from 'react';
+import { BentoFacebook } from '@ampproject/bento-facebook/react';
+import '@ampproject/bento-facebook/styles.css';
+
+function App() {
+  return (
+    <BentoFacebook
+        embedAs="post"
+        href="https://www.facebook.com/NASA/photos/a.67899501771/10159193669016772/">
+    </BentoFacebook>
+  );
+}
+```
+
+[/example]
+
+#### Props
+
+##### `href`
+
+The URL of the Facebook post/video/comment. For example, a post or video will
+look like `https://www.facebook.com/zuck/posts/10102593740125791`. A comment or
+comment reply will look like
+`https://www.facebook.com/zuck/posts/10102735452532991?comment_id=1070233703036185`.
+For comments, see the Facebook documentation on
+[how to get a comment's URL](https://developers.facebook.com/docs/plugins/embedded-comments#how-to-get-a-comments-url).
+
+##### `embedAs`
+
+The value is either `post`, `video` or `comment`. The default is `post`.
+
+Both posts and videos can be embedded as a post. Setting `data-embed-as="video"`
+for Facebook videos embeds the player of the video, and adds the accompanying
+post card with it. Setting `data-embed-as="post"` ignores the caption card. This
+is done to make sure we are zooming in on videos correctly.
+
+The `comment` value embeds a single comment (or reply to a comment) on a post.
+This is not to be confused with
+[`amp-facebook-comments`](https://amp.dev/documentation/examples/components/amp-facebook-comments/).
+
+Check out the documentation for differences between
+[post embeds](https://developers.facebook.com/docs/plugins/embedded-posts),
+[video embeds](https://developers.facebook.com/docs/plugins/embedded-video-player),
+and [comment embeds](https://developers.facebook.com/docs/plugins/embedded-comments).
+
+##### `includeCommentParent`
+
+The value is either `true` or `false`. The default is `false`.
+
+When you are embedding a comment reply, you can optionally also include the
+parent comment of the reply.
+
+##### `allowfullscreen`
```

Comment:
This should be [`allowFullScreen`](https://github.com/facebook/react/blob/cae635054e17a6f107a39d328649137b83f25972/packages/react-dom/src/shared/possibleStandardNames.js#L18)
```suggestion
##### `allowFullScreen`
```